loading.bs

```diff
@@ -988,6 +1053,44 @@ A [=request=] |browserRequest| <dfn>matches the stored exchange</dfn>
     Behavior=] returning a list of lists.
 1. Return "match".
 
+<h3 algorithm id="queue-report">Queuing signed exchange report</h3>
+
+<dfn>Queuing signed exchange report</dfn> |report| is the following steps:
+
+1. Let |report body| and |policy| be the result of
+    [=generate a network error report=]</a> with |report|'s
+    [=signed exchange report/outer request=]. If the result is null, abort
+    these steps.
+
+1. If |report body|'s `"type"` is `"dns.address_changed"`, abort these steps.
+
+1. Add a new property `"sxg"` to |report body| with a new ECMAScript object with
+     the following properties:
+
+    * outer_url: |report|'s [=signed exchange report/outer request=]'s
+          [=request/url=].
+    * inner_url: |report|'s [=signed exchange report/inner URL=].
+    * cert_url: |report|'s [=signed exchange report/cert URL=].
+
+1. Set |report body|'s `"phase"` to `"sxg"`.
+
+1. Set |report body|'s `"type"` to the result of concatenating a string `"sxg."`
+     and the |report|'s [=signed exchange report/result=].
+
+1. If |report body|'s `"type"` is not `"sxg.ok"` and |report body|'s `"sxg"`'s
+    `"cert_url"`'s [=url/scheme=] is not `"data"`:
+
+    1. If |report body|'s `"sxg"`'s `"outer_url"`'s [=url/origin=] is different
+        from |report body|'s `"sxg"`'s `"cert_url"`'s [=url/origin=] or
+        |report|'s [=signed exchange report/server IP=] is different from
+        |report|'s [=signed exchange report/cert server IP=]:
+
+        1. Set |report body|'s `"type"` to `"sxg.failed"`.
```

Comment:
Do we need to block all errors when the certificate comes from a different IP than the SXG or just certificate errors? Add a `Note:` paragraph (or a privacy consideration like in https://w3c.github.io/network-error-logging/#privacy-considerations) saying why we block whatever we block.